### PR TITLE
Update run-a-message-trace-and-view-results.md

### PR DESCRIPTION
**Pull request by kellybos**
Updated information about DLP rule and sensitivity label events. These are now separate and it's important to make clear why type of sensitivity label events are included. These are EXO events only so are not included in the Exchange server location that is linked to later in the article.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/monitoring/trace-an-email-message/run-a-message-trace-and-view-results.md [hunk 1]
```diff
@@ -130,7 +130,8 @@ In the events section, the following fields provide information about the events
   - **TRANSFER**: Recipients were moved to a bifurcated message because of content conversion, message recipient limits, or agents.
   - **DEFER**: The message delivery was postponed and may be re-attempted later.
   - **RESOLVED**: The message was redirected to a new recipient address based on an Active Directory look up. When this happens, the original recipient address is listed in a separate row in the message trace along with the final delivery status for the message.
-  - **DLP rule**: The message had a DLP rule or sensitivity label match in this message.
+  - **DLP rule**: The message had a DLP rule match in this message.
+  - **Sensitivity label:** A server side labeling event occurred. For example, a label automatically added to a message that includes an action to encrypt or added via the web or mobile client. This action is completed by the Exchange server and logged. A label added via Outlook will not be included in the event field.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/monitoring/trace-an-email-message/run-a-message-trace-and-view-results.md:134]:
```suggestion
  - **Sensitivity label:** A server-side labeling event occurred, such as a label automatically added to a message that includes an action to encrypt or added via the web or mobile client. This action is completed by the Exchange server and logged. A label added via Outlook will not be included in the event field.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kellybos, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@JoanneHendrickson @chrisda 
Can you review?
Thanks!

**Pull request merged**